site/content/docs/0.2/extend/icons.md

```diff
@@ -1,11 +1,736 @@
+      </svg> Info
+    </p>
+    <p>
+      <svg class="text-status-positive decorative-sm-icon" aria-hidden="true">
+        <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#tick-confirmation"/>
+      </svg> Success
+    </p>
+    <p>
+      <svg class="text-status-warning decorative-sm-icon" aria-hidden="true">
+        <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#important"/>
+      </svg> Warning
+    </p>
+    <p>
+      <svg class="text-status-negative decorative-sm-icon" aria-hidden="true">
+        <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#error-severe"/>
+      </svg> Danger
+    </p>
+  </div>
+  <div class="p-medium bg-emphasized border border-default" data-bs-theme="dark">
+    <p>
+      <svg class="decorative-sm-icon" style="color: var(--bs-color-content-status-info)" aria-hidden="true">
+        <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#info"/>
+      </svg> Info
+    </p>
+    <p>
+      <svg class="decorative-sm-icon" style="color: var(--bs-color-content-status-positive)" aria-hidden="true">
+        <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#tick-confirmation"/>
+      </svg> Success
+    </p>
+    <p>
+      <svg class="decorative-sm-icon" style="color: var(--bs-color-content-status-warning)" aria-hidden="true">
+        <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#important"/>
+      </svg> Warning
+    </p>
+    <p>
+      <svg class="decorative-sm-icon" style="color: var(--bs-color-content-status-negative)" aria-hidden="true">
+        <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#error-severe"/>
+      </svg> Danger
+    </p>
+  </div>
+</div>
+{{< /example >}}
+
+Here are [the normal contexts of use of these functional colors](https://unified-design-system.orange.com/472794e18/p/217ac6-colour/t/9b95834c2e).
+
+{{< bs-table >}}
+| Color utility | Color CSS variable | Context of use |
+| --- | --- | --- |
+| `.text-status-info` | `--bs-color-content-status-info` | Use to communicate a feedback. It's more important than a neutral color. |
+| `.text-status-positive` | `--bs-color-content-status-positive` | Use to communicate a positive action, a confirmation or a positive feedback. |
+| `.text-status-warning` | `--bs-color-content-status-warning` | Use to display an information that requires more attention, or an action from the user. |
+| `.text-status-negative` | `--bs-color-content-status-negative` | Use to communicate something negative. It can be a destructive action, an error state, or a negative feedback. |
+{{< /bs-table >}}
+
+### Other icons
+
+Here are some examples of other [color utilities]({{< docsref "utilities/colors#colors" >}}) and CSS variables used with icons.
```

Comment:
Should we add a table like the one above ?

Reply:
I could, but I don't have the context of usage...